docs/source/blocks/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 myst:
   html_meta:
-    "description": "Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks."
-    "property=og:description": "Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks."
-    "property=og:title": "Blocks"
-    "keywords": "Volto, Plone, frontend, React, blocks"
+    'description': 'Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks.'
```

Comment:
@erral @ksuess @stevepiercy uh I realised we are using different types of quotes here, also in next one...

Is there any accepted styling for md meta?

Reply:
Double quotes `"` is what is created initially by a Python script, and can be copy-pasted from other .md files. This is consistent with Black and other Python style guides for Plone. If single quotes are being used, we need to change them.

Also I should review this documentation before merge. Please give me until tonight to do that. Thank you!